iree/compiler/Dialect/HAL/Conversion/FlowToHAL/ConvertStreamOps.cpp

```diff
@@ -940,10 +940,15 @@ static LogicalResult recordDispatch(Value device, Value commandBuffer,
   return success();
 }
 
-// Splats a pattern value of 1, 2, or 4 bytes out to a 4 byte value.
+// Splats a pattern value of 1, 2, or 4 bytes out to a 4 byte integer value.
```

Comment:
I think this could use more explanation as to what it's doing. It creates a 4 byte value that is byte-equivalent to n times the passed in value. Took me a while to figure that out.